cmake/alpakaCommon.cmake

```diff
@@ -855,11 +893,28 @@ if((ALPAKA_ACC_GPU_CUDA_ENABLE OR ALPAKA_ACC_GPU_HIP_ENABLE) AND ALPAKA_CUDA_COM
                  TARGET alpaka
                  PROPERTY INTERFACE_COMPILE_OPTIONS)
     string(REPLACE ";" " " _ALPAKA_COMPILE_OPTIONS_STRING "${_ALPAKA_COMPILE_OPTIONS_PUBLIC}")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_ALPAKA_COMPILE_OPTIONS_STRING}")
+    alpaka_set_compiler_options(HOST set CMAKE_CXX_FLAGS "${_ALPAKA_COMPILE_OPTIONS_STRING}")
```

Comment:
Why would `alpaka_set_compiler_options(HOST list CMAKE_CXX_FLAGS ${_ALPAKA_COMPILE_OPTIONS_STRING})` be wrong here?

Reply:
`CMAKE_CXX_FLAGS` is not a list. It is a string with space-separated entries.

Reply:
The new command is now `alpaka_set_compiler_options(HOST var CMAKE_CXX_FLAGS ${_ALPAKA_COMPILE_OPTIONS_STRING})`

Documentation of `var` is: `var: is spaces sperated`

Reply:
OK!